.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Cli Release
+on:
+  workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog, Please provide as a string.'
+        required: true
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout to cli
```

Comment:
* "Checkout cli" ?

Reply:
Yes, the below action `actions/checkout@v2` checks-out cli repository under `$GITHUB_WORKSPACE`, so this workflow can access it.

Reply:
I meant, isn't it a typo: Shouldn't it be "Checkout cli" ?

Reply:
Ohh, yeah. Will make change.